### ISSUE TITLE
Should wcwidth provide rjust, ljust, and center?

### ISSUE THREAD

**Issue by jquast** [jquast/wcwidth#93]
In the readme, we talk about the example of writing our own custom rjust function. If the most common use case is text alignment, let's just offer it as a public API. They would be the same name and signature as python's,

- https://docs.python.org/3/library/stdtypes.html#str.ljust
- https://docs.python.org/3/library/stdtypes.html#str.lstrip
- https://docs.python.org/3/library/stdtypes.html#str.center

**Comment by dscrofts** [comment]
Definitely! I have implemented the `ljust` function as per the readme but it would be super convenient to have this already included.

**Comment by jquast** [comment]
On textwrap, there is a pretty strange edge-case found in downstream blessed library https://github.com/jquast/blessed/pull/275 -- what should be done when wrapping wide characters to a column length of 1? We have no choice but to flow beyond column width. A width of "1" may not even be allowed, or allowed with caveats.

We must also be very careful not to split ZWJ and VS-16 sequences apart.
